repos/fdbt-site/src/pages/multiOperatorServiceList.tsx

```diff
@@ -466,12 +464,12 @@ export const getServerSideProps = async (
         multiOperatorData = await Promise.all(
             ticket.additionalOperators.map(async (operator) => {
                 let dataSource: 'bods' | 'tnds' = 'bods';
-                let services = await getServicesByNocCodeAndDataSourceWithGrouping(operator.nocCode, dataSource);
 
-                if (services.length === 0) {
-                    services = await getFerryAndBusServices(operator.nocCode);
+                if (modesAttribute) {
                     dataSource = 'tnds';
                 }
```

Comment:
I dont think this is correct because if the user is logged in as a bus operator, they can no longer add Tram operators to their group. They wont have the modesAttribute in their session. I don't believe this page needs to change at all.